test/unit/renderer/dataframe.test.js

```diff
@@ -1,9 +1,18 @@
 import Dataframe from '../../../src/renderer/Dataframe';
 import Metadata from '../../../src/renderer/Metadata';
 import { pointInTriangle } from '../../../src/utils/geometry';
-import { translate } from '../../../src/renderer/viz/expressions';
+import { mat4 } from 'gl-matrix';
 
 describe('src/renderer/Dataframe', () => {
+    const m = [];
```

Comment:
Here we could reuse `_createMatrix ` method from Map.js (after just a little refactor to receive ratio as a parameter).

Reply:
Yes... but re-using that complex logic from the implementation in the tests... I'd prefer to keep this as is to be honest

Reply:
No-worries man